library/core/src/main/java/com/google/android/exoplayer2/mediacodec/MediaCodecRenderer.java

```diff
@@ -2478,6 +2486,13 @@ private static boolean codecNeedsMonoChannelCountWorkaround(String name, Format
         && "OMX.MTK.AUDIO.DECODER.MP3".equals(name);
   }
 
+  private static boolean codecNeedsSkipSurplusOutBufferWorkaround(String name, Format format) {
+    return Util.SDK_INT >= 29
+        && format.width >= 1920
+        && format.height >= 1080
+        && "OMX.amlogic.avc.decoder.awesome2".equals(name);
+  }
```

Comment:
Is it all 1920x1080 content that frame rate conversion happens for?  What about other 30FPS content (4k?) on a 60HZ refresh TV?